### PR TITLE
Updated wcslib to latest version

### PR DESCRIPTION
**Pull request by rlancaste**
### Have you:

- [x ] Followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-science/blob/master/.github/CONTRIBUTING.md) document?
- [ x] Checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-science/pulls) for the same update/change?
- [ x] Run `brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting) and corrected all errors?
- [x ] Built your formula locally prior to submission with `brew install <formula>`?

### New formula: have you

- [x ] Written a sensible test? (The best test is to compile and run a sample program.)

### Updates to existing formula: have you

- [ ?] Removed the `revision` line, if any, when bumping the version number?
- [ ?] Added/bumped the `revision` number if the changes affect the precompiled bottles?
- [ x] Not altered the `bottle` section?
- [ x] Checked that the tests still pass?

### CODE REVIEW

File: wcslib.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class Wcslib < Formula
   homepage "http://www.atnf.csiro.au/people/mcalabre/WCS/"
-  url "ftp://ftp.atnf.csiro.au/pub/software/wcslib/wcslib-5.12.tar.bz2"
-  sha256 "50ff182920541bea156e7f53588cdfba1754b1bdcd0bc64a136a5d124af98de4"
+  url "ftp://ftp.atnf.csiro.au/pub/software/wcslib/wcslib-5.16.tar.bz2"
+  sha256 "ed031e0cf1cec0e9cabfc650423efa526fec341441865001c1e2c56bfffc99ef"
   revision 1
```
Comment (iMichka) [inline wcslib.rb]: Please remove that line (revision 1) completely when bumping the version number.

**Comment by iMichka** [comment]
Hi. Thanks for your contribution. This looks already very good.

I made a small comment about the `revision 1` line that needs to be removed. Could you also please update the commit message so that the title (first line) is: `wcslib 5.16` (as per the contributing guidelines).

You can make your changes, use `git commit --amend` to update the commit content and message, and push to Github with `git push --force`. That should update update the pull request automatically here.

For cross ref, this is related to #4856

**Comment by rlancaste** [comment]
I attempted to follow your instructions, but it only changed the message not the removal of the line, so I had to do it again with a separate commit.  I hope it worked.

**Comment by iMichka** [comment]
It's ok. I will squash the commits myself before merging. Let's wait for the builds to return to see if all the tests pass and if bottles are built. If everything is green, I (or somebody else) will merge this tomorrow.

**Comment by iMichka** [comment]
@rlancaste thanks for your contribution to Homebrew.

**Pull request closed without merging**